acceptance-test/user-pathways/happy-path/happy-path.test.js

```diff
@@ -170,7 +191,41 @@ describe.only('User path(s)', () => {
       await clickSelector(page, CONTINUE_BUTTON);
     }
 
-    await focusThenType(page, WHAT_HAPPENED_INPUT, 'Test input regarding details of exploitation');
+    await focusThenType(page, PV_BIRTHPLACE, 'Test input of birthplace');
+    await focusThenType(page, PV_FAMILY, 'Test input of familty');
+    await focusThenType(page, PV_EDUCATION, 'Test input of education');
+    await focusThenType(page, PV_EMPLOYMENT_HISTORY, 'Test input of employment history');
+    await clickSelector(page, CONTINUE_BUTTON);
+    await focusThenType(page, EXPLOITATION_TAKE_PLACE, 'Test input of exploitation location');
+    await clickSelector(page, CONTINUE_BUTTON);
+    await focusThenType(page, MORE_THAN_ONE_SITUATION, 'Test input of more than on situation');
+    await clickSelector(page, CONTINUE_BUTTON);
+    await focusThenType(page, HOW_DID_EXPLOITATION_START, 'Test input of start date');
+    await clickSelector(page, CONTINUE_BUTTON);
+    await clickSelector(page, WERE_TAKEN_SOMEWHERE_YES);
+    await focusThenType(page, WERE_TAKEN_SOMEWHERE_DETAILS, 'Test input of location details');
+    await clickSelector(page, CONTINUE_BUTTON);
+    await focusThenType(page, WHAT_WERE_THEY_REQUIRED_TO_DO, 'Test input of what they were required to do');
+    await focusThenType(page, HOW_WERE_THEY_TREATED, 'Test input of how they were treated');
+    await focusThenType(page, WHY_THEY_STAYED, 'Test input of why they stayed');
+    await clickSelector(page, CONTINUE_BUTTON);
+    await focusThenType(page, WHY_DID_THEY_LEAVE, 'Test input of why they left');
+    await clickSelector(page, CONTINUE_BUTTON);
+    await clickSelector(page, FIRST_CHANCE_TO_REPORT_YES);
+    await clickSelector(page, CONTINUE_BUTTON);
+    await focusThenType(page, WHY_REPORT_NOW, 'Test input of why report now');
+    await clickSelector(page, CONTINUE_BUTTON);
+    await focusThenType(page, WHY_ARE_YOU_MAKING_REFERRAL, 'Test input of reason for referral');
+    await clickSelector(page, CONTINUE_BUTTON);
+    await focusThenType(page, INTERVIEW_CARRIED_OUT, 'Test input of interview information');
+    await clickSelector(page, CONTINUE_BUTTON);
+    await clickSelector(page, ARE_OTHERS_INVOLVED_YES);
+    await focusThenType(page, ARE_OTHERS_INVOLVED_DETAILS, 'Test input of other involved details');
+    await clickSelector(page, CONTINUE_BUTTON);
+    await clickSelector(page, EVIDENCE_OF_DISHONESTY_YES);
+    await focusThenType(page, EVIDENCE_OF_DISHONESTY_DETAILS, 'Test input of evidence of dishonesty details');
+    await clickSelector(page, CONTINUE_BUTTON);
+    await focusThenType(page, EVIDENCE_TO_SUBMIT, 'Test input of evidence to submit');
```

Comment:
I'm guess you need these because of all the new pages? it's like you are adding lots of new tests

Reply:
yes, all the new pages are needed so the form can go through the tests. These ones are linear from beginning to end through the form.